Google Cloud Translation API integration

aerjotl opened the pull request:
I needed to translate my website to a new language, which was not supported by Deepl. Free Google API was constantly giving timeouts, so I implemented Google Cloud API plugin, which worked great and by the way, it's also free up to a certain usage threshold.

ben182 commented:
Hi thanks for the PR! Really appreciate it. May you please write an additional paragraph in the readme for the price? You wrote Free: "Yes*". But what does that asterisk mean?

aerjotl commented:
hi @ben182, their pricing scheme is not totally clear for me. 
They charge per character. I translated around 1000 messages and I paid for it 0,44€.

you can find the pricing details:
https://cloud.google.com/translate/pricing

I updated the readme and I replaced Free: "Yes*" with "No".
